relay-server/src/services/processor/event.rs

```diff
@@ -145,10 +149,37 @@ pub fn extract<Group: EventProcessing>(
 
     metrics.bytes_ingested_event = Annotated::new(event_len as u64);
 
+    if let Some(event_value) = event.value() {
+        let convert_breadcrumbs_to_logs = utils::sample(
+            global_config
+                .options
+                .ourlogs_breadcrumb_extraction_sample_rate
+                .unwrap_or(0.0),
+        );
+
+        if convert_breadcrumbs_to_logs {
+            relay_log::trace!("extracting breadcrumbs to logs");
+            let ourlogs: Vec<relay_event_schema::protocol::OurLog> =
```

Comment:
Does the compiler need this type annotation?

```suggestion
            let ourlogs =
```